db/warehouse/migrate/20221116185411_add_index_to_census.rb

```diff
@@ -0,0 +1,7 @@
+class AddIndexToCensus < ActiveRecord::Migration[6.1]
+  def change
+    add_index :nightly_census_by_projects, :date
+    add_index :nightly_census_by_projects, :project_id
+    add_index :nightly_census_by_project_types, :date
```

Comment:
Can delete from `nightly_census_by_project_types` since we don't use that table anymore. May need more indices on project table?